### PR TITLE
Feature/issue 139

### PR DESCRIPTION
**Pull request by dieppa**

### CODE REVIEW

File: dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 public interface DimmerConfigReader {
     DimmerConfig loadConfiguration() throws DimmerConfigException;
-    Map.Entry<String, EnvironmentConfig> getDefaultEnvironment(DimmerConfig dimmerConfig);
+    @Deprecated
```
Comment (dhivesh) [inline dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java]: Delete as it has been refactored. The only module using this is dimmer-local module.

File: dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java [hunk 2]
```diff
@@ -8,6 +8,10 @@
...
+    EnvironmentConfig getEnvironment(DimmerConfig dimmerConfig, String environment);
+    @Deprecated
```
Comment (dhivesh) [inline dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java]: Same as above

File: dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java (hunk 1 above)
Comment (dieppa) [reply dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java]: 👍

File: dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java (hunk 2 above)
Comment (dieppa) [reply dimmer-core/src/main/java/com/github/cloudyrock/dimmer/reader/DimmerConfigReader.java]: 👍

**Pull request merged**